Description: Add -Winvalid-pch to CFLAGS.
 This will allow a more verbose build, which may help us to determine the reason
 of the FTBFS on ia64.
Author: Stephan Schreiber <info@fs-driver.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=696096

--- a/mkspecs/common/g++-base.conf.orig	2012-12-16 14:36:37.347714646 +0100
+++ b/mkspecs/common/g++-base.conf	2012-12-16 14:37:45.335717820 +0100
@@ -25,6 +25,6 @@
 QMAKE_PCH_OUTPUT_EXT = .gch
 
 QMAKE_CFLAGS_PRECOMPILE       = -x c-header -c ${QMAKE_PCH_INPUT} -o ${QMAKE_PCH_OUTPUT}
-QMAKE_CFLAGS_USE_PRECOMPILE   = -include ${QMAKE_PCH_OUTPUT_BASE}
+QMAKE_CFLAGS_USE_PRECOMPILE   = -include ${QMAKE_PCH_OUTPUT_BASE} -Winvalid-pch
 QMAKE_CXXFLAGS_PRECOMPILE     = -x c++-header -c ${QMAKE_PCH_INPUT} -o ${QMAKE_PCH_OUTPUT}
 QMAKE_CXXFLAGS_USE_PRECOMPILE = $$QMAKE_CFLAGS_USE_PRECOMPILE
